src/converters/WmsCatalogGroup.ts

```diff
@@ -11,6 +11,8 @@ import {
   getUnknownProps,
   itemProperties,
   propsToWarnings,
+  catalogGroupPropsIgnore,
+  catalogGroupProps,
 } from "./helpers";
```

Comment:
Can we remove the now unused imports here - i.e `catalogMemberPropsIgnore` and `catalogMemberProps`